src/api/ledger/trustlines.js

```diff
@@ -56,7 +53,7 @@ function getTrustLines(account, options, callback) {
         ledger = prevResult.ledger_index;
       } else {
         marker = options.marker;
-        limit = validator.isValid(options.limit, 'UINT32')
+        limit = parseInt(options.limit, 10) === options.limit
           ? Number(options.limit) : DefaultPageLimit;
```

Comment:
I'll get rid of this entirely by using the convention that no limit specified means "all"